api/src/main/java/com/epam/pipeline/manager/region/AzureRegionHelper.java

```diff
@@ -141,27 +131,17 @@ public String serializeCredentials(AzureRegion region, AzureRegionCredentials cr
         }
     }
 
-    private void validateStorageAccount(final String storageAccountName, final String storageAccountKey) {
-        Assert.isTrue(StringUtils.isNotBlank(storageAccountName),
```

Comment:
I think that we should leave such checks, but make it conditional depending on region settings